bpfman/src/command.rs

```diff
@@ -36,6 +36,62 @@ use crate::{
     ROOT_DB,
 };
 
+const SLED_DB_KEY_KIND: &str = "kind";
```

Comment:
so this looks alright but can you please just remove the `SLED_DB_KEY` prefixes from all these variable names in favor of just a code comment above the block where the keys are defined?

Reply:
done